2.13/2.13.1/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.13.1
+При помощи цикла for выведите чётные числа от 2 до 10.
+<script>
+	for (let i = 2; i < 11; i++) {
+		if (i % 2 === 0) {
+			alert(`${i}`);
```

Comment:
there isn't any static message in this alert, so you don't need to use the pattern with back quotes